src/styles/_products.scss

```diff
@@ -0,0 +1,27 @@
+.products {
+  &__list {
+    list-style: none;
+    display: grid;
+    row-gap: 46px;
+    column-gap: 48px;
+    justify-content: center;
+    padding: 50px 40px 128px;
+    grid-template-areas: "row";
+
+    @media screen and (min-width: 488px) {
+      grid-template-areas: "row row";
+    }
+
+    @media screen and (min-width: 768px) {
+      grid-template-areas: "row row row";
+    }
+
+    @media screen and (min-width: 1024px) {
+      grid-template-areas: "row row row row";
+    }
```

Comment:
Why do you call them "row" but in our case they are columns?😁
With grid-template-columns: repeat("number of columns", "column width"); it's easier to manage in this case.